lib/global_config.d.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+
```

Comment:
Unnecessary line

Reply:
I deleted the global_config.d.ts file from the PR because the user does not need to use these options.